patches/tModLoader/Terraria/WorldGen.cs.patch

```diff
@@ -1475,7 +2672,7 @@
 +			return result;
 +		}
 -		public static int KillTile_GetTileDustAmount(bool fail, Tile tileCache) {
-+		private static int KillTile_GetTileDustAmount_Vanilla(bool fail, Tile tileCache) {
++		public static int KillTile_GetTileDustAmount_Vanilla(bool fail, Tile tileCache) {
```

Comment:
No need for making a split public.

Reply:
I just regexed all of this. I can go through and modify stuff that doesn't necessarily need to be public.